pkg/proto/configuration/jwt/jwt.proto

```diff
@@ -14,17 +15,31 @@ message AuthorizationHeaderParserConfiguration {
   // Was 'public_key'. Replaced by 'jwks_inline'.
   reserved 2;
 
-  // A JSON Web Key Set (JWKS) as described in RFC 7517, chapter 5,
-  // containing one or more public keys against which tokens signatures
-  // are validated.
-  //
-  // Public keys encoded in DER or PEM format can be converted to a JSON
-  // Web Key Set using one of the following tools:
-  //
-  // - https://smallstep.com/docs/step-cli/reference/crypto/key/format/
-  // - https://irrte.ch/jwt-js-decode/pem2jwk.html
-  // - https://russelldavies.github.io/jwk-creator/
-  google.protobuf.Struct jwks_inline = 7;
+  message JwksFile {
+    // Path to JWKS file.
+    string file_path = 1;
+
+    // Interval at which to refresh JWKS file.
+    google.protobuf.Duration refresh_interval = 2;
```

Comment:
Maybe having a refresh interval adds too much flexibility. Is there some kind of constant that we can use here that tends to be sufficient for most use cases? 5 minutes? If so, we should just use that.

Reply:
If I we're gonna set a static value, 300 seconds would have been my choice as well

Reply:
Since we're not keeping this field, I've turned `jwks_file` into a string type. The refresh rate is a static 300 seconds.